service/src/main/java/com/ge/predix/acs/privilege/management/dao/ResourceRepositoryProxy.java

```diff
@@ -26,10 +26,18 @@
     @Autowired(required = false)
     private GraphResourceRepository graphRepository;
 
+    public GraphResourceRepository getGraphRepository() {
```

Comment:
Valid point about changing this method from being public. So we've made it package private. 

Also, we wanted to keep the proxy class as a single entry-point for all repo related queries. So we avoided auto-wiring 4 individual repos in this class and use the proxy.  We can discuss this